cdbmss.c

```diff
@@ -34,7 +34,7 @@ unsigned int datalen;
   for (i = 0;i < keylen;++i)
     h = cdbmake_hashadd(h,(unsigned int) key[i]);
 
-  if (!cdbmake_add(&c->cdbm,h,c->pos,alloc)) return -1;
+  if (!cdbmake_add(&c->cdbm,h,c->pos,malloc)) return -1;
```

Comment:
In case we want to keep the source entirely untouched, we may use
```C
#define alloc malloc
``` 
But I am also happy to remove residues of `alloc` when possible.

Reply:
+1 to keep referring to `alloc` here, since we can.

Reply:
We would add a #define for only one line of code that no patch will likely ever touch. I don't see a benefit in this.

Reply:
done